dist/settings.json

```
@@ -379,6 +379,12 @@
       "Commands": [
         "mute"
       ],
+    },
+    {
+      "CommandName": "SetJob",
+      "Commands": [
+        "setjob"
+      ],
```

```suggestion
      ]
```
Remove the trailing comma

---

already fixed